engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -13,9 +15,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends WorkflowStore {
   override def initialize(implicit ec: ExecutionContext): Future[Unit] = {
-    sqlDatabase.updateWorkflowState(
-      WorkflowStoreState.Running.toString,
-      WorkflowStoreState.Restartable.toString)
+          if ( ConfigFactory.load().as[Option[Boolean]]("system.workflow-restart").getOrElse(true)) {
+            sqlDatabase.updateWorkflowState(
+                WorkflowStoreState.Running.toString,
+                WorkflowStoreState.Restartable.toString)
+          } else {
+            Future.successful(())
```

Comment:
Will they stay forever in the WorkflowStore with a "Running" state in this case ?